pkg/api/types.go

```diff
@@ -0,0 +1,44 @@
+package api
+
+import (
+	docker "github.com/fsouza/go-dockerclient"
+	"time"
+)
+
+// OpenSCAPStatus is the status of openscap scan
+type OpenSCAPStatus string
+
+const (
+	StatusNotRequested OpenSCAPStatus = "NotRequested"
+	StatusSuccess      OpenSCAPStatus = "Success"
+	StatusError        OpenSCAPStatus = "Error"
+)
+
+type OpenSCAPMetadata struct {
+	Status           OpenSCAPStatus // Status of the OpenSCAP scan report
+	ErrorMessage     string         // Error message from the openscap
+	ContentTimeStamp string         // Timestamp for this data
```

Comment:
Why is this a string?  Would there be a benefit to leaving it as `Time`?  If it is because of issues with json marshalling should we take the Kube approach and provide a wrapper (see `unversioned.Time` in Kube) so we retain the option to perform time based functions on this field?